js/contact.js

```diff
@@ -0,0 +1,61 @@
+// Your code goes here
+window.addEventListener('load', (event) => {
```

Comment:
this is legitimate code, but you can make it more specific and say `document.addEventListener` instead of window. WIth window you are referencing everything